packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -9,9 +9,6 @@ import {
   makeOnewayPriceAuthorityKit,
 } from '../contractSupport';
 
-import '../../exported';
-import './exported';
-
```

Comment:
I think this should be: `import '../../exported';`

Reply:
`yarn lint` and vsCode are happy without it.